docs/how-to/code/use-django-extension/example/example_django/example_django/asgi.py

```diff
@@ -0,0 +1,16 @@
+"""
+ASGI config for example_django project.
+
+It exposes the ASGI callable as a module-level variable named ``application``.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/5.0/howto/deployment/asgi/
+"""
+
+import os
+
+from django.core.asgi import get_asgi_application
+
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "example_django.settings")
```

Comment:
Is this the preferred way to do this? It seems to me like putting this inside the rock's environment would be cleaner (though IDK what downsides it would have).

Reply:
Yes, all Django projects in the tests are generated using the Django admin tool, so I think it's okay.